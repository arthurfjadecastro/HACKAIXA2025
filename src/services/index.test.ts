import * as servicesIndex from './index';

describe('Services Index', () => {
  it('should export products services', () => {
    expect(servicesIndex).toBeDefined();
    expect(typeof servicesIndex).toBe('object');
  });

  it('should re-export modules correctly', () => {
    // Verifica se o mÃ³dulo foi exportado sem erros
    expect(() => {
      const exported = { ...servicesIndex };
      expect(exported).toBeDefined();
    }).not.toThrow();
  });

  it('should match snapshot', () => {
    expect(Object.keys(servicesIndex).sort()).toMatchSnapshot();
  });
});
