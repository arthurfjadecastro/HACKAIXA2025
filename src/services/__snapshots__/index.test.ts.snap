// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Services Index should match snapshot 1`] = `
[
  "ProductDataService",
  "calculateLoanSchedule",
  "convertMonthlyToYearlyRate",
  "convertYearlyToMonthlyRate",
  "productService",
  "simulationsApi",
]
`;
